frontend/flashfolio/src/Viewer.js

```diff
@@ -145,6 +156,7 @@ export default function Viewer({ viewMode = "view" }) {
 			Title: {flashdeck.current.Title}
 			DeckId: {deckId}
 			<br />
+			<button onClick = {flipView}> {viewMode == "edit" ? <a>View Deck</a> : <a>Edit Deck</a>} </button>
```

Comment:
You don't need to wrap the interior text in `<a>` tags. Just have the ternary operator return strings. By the way it is set up right now the button looks kinda silly -- see how it looks like there is a button inside a button?

![Screenshot from 2021-11-11 12-34-43](https://user-images.githubusercontent.com/78443917/141343538-dcbf359f-c8d5-458a-839c-31060db2cdf9.png)